dayFive/changeMax.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = function(currentMin : number, currentMax : number) : number {
+  return currentMax - (Math.floor((currentMax - currentMin) / 2)) - 1;
+}
```

Comment:
This function is small enough that it should probably be defined in the file in which it's used